### ISSUE TITLE
perTouch has no effect

### ISSUE THREAD

**Issue by espenlg** [glidejs/glide#624]
No matter what I set the perTouch value to it always moves 1 image when swiping (at least in Chrome).
Same thing when I try on https://glidejs.com/docs/options/#pertouch and adjust the setting. It does not change anything?

**Comment by jedrzejchalubek** [comment]
Definitely, something is wrong there. Not sure in which version it appeared need to investigate it

**Comment by Raidri** [comment]
In 3.4.1 the perTouch works as expected since 3.5.0 it's broken.

**Comment by ipyguru** [comment]
In 3.4.1 the perTouch dont working

**Comment by wsSteve** [comment]
As of version 3.5.2, perTouch is still broken. Any chance of this getting fixed in a future release?

**Comment by jeffdrumgod** [comment]
in the document there is `perTouch`, but not in the code. Use `perSwipe`, which does something similar.

https://github.com/glidejs/glide/blob/master/src/defaults.js#L99
